.kokoro/nightly/downstream-protobuf-source-compatibility.sh

```diff
@@ -28,11 +28,18 @@ for repo in ${REPOS_UNDER_TEST//,/ }; do # Split on comma
   git clone "https://github.com/googleapis/$repo.git" --depth=1
   pushd "$repo"
 
+  # Compile with Java 11 and run the tests with Java 8
+  mvn clean compile -T 1C
+
+  export JAVA_HOME="${JAVA8_HOME}"
+  export PATH=$JAVA_HOME/bin:$PATH
+  java -version
```

Comment:
Maven's "-V" shows the version too.

Reply:
Oh I left this in. I'll remove this version check since I've confirmed from the logs that it is switching versions properly.